README.md

```diff
@@ -10,15 +10,43 @@ The purpose of this repository is to allow easy management of all Pop!\_OS relat
 
 Binary packages are hosted [on Launchpad](https://launchpad.net/~system76/+archive/ubuntu/pop/+packages). Many packages have source on Github, under the [Pop!\_OS organization](https://github.com/pop-os).
 
+## Developer Resources
+For instructions on how to build the shell:
+	* [Cosmic, our current gnome-based shell](https://github.com/pop-os/cosmic). 
```

Comment:
This bullet-point list isn't displaying properly in the GitHub viewer, probably because it's indented too far.

COSMIC should be stylized in all caps (it is technically an acronym.) I would not use the word "our" like this. I would make the link text something shorter and more consistent like `COSMIC (GNOME-based)` and `COSMIC Epoch (Rust-based)`. Remove the periods from the end of the list items.